### PR TITLE
Remove bottom padding between content and major links section

### PR DESCRIPTION
**Pull request by aubreyarcangel**
## Description
This PR removes the bottom padding from content pages when frontmatter property`no_article_bottom_padding` is set to `true`.

A conditional statement was added within the `src/site/includes/detail-page.html` page layout. When the frontmatter property`no_article_bottom_padding` is set to `true` on a content page, the conditional will add a spacing utility `vads-u-padding-bottom--0`, and remove the default `padding-bottom: 3em` in the `article` class.

Note: When viewing this PR, it should be paired with [this vagov-content PR](https://github.com/department-of-veterans-affairs/vagov-content/pull/307)

and on this url: /health-care/health-needs-conditions/health-issues-related-to-service-era/

## Testing done
Viewed on local build

## Screenshots
**before**:
Screen Shot 2019-03-13 at 3 18 18 PM

**after**:
Screen Shot 2019-03-13 at 3 18 36 PM

## Acceptance criteria
- [X] bottom padding is removed when flag is set to `true`

## Definition of done
- [X] Events are logged appropriately
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by ncksllvn** [comment]
Was this issue occurring on just this single page? Or a series of pages? I'm thinking right now -

1. If it's just this one page, I would expect an issue with the HTML or some weird CSS.
2. If it's a series of pages, can we just apply this rule to all of them and not worry about the a new front matter property?

**Comment by aubreyarcangel** [comment]
@ncksllvn by default, the `padding-bottom: 3em;` is applied to all content pages defined in `/src/platform/site-wide/sass/merger.scss` on line 59

```
  article {
    &:last-of-type {
      padding-bottom: 3em;
    }
  }
```
I think the padding looks excessive here because there is no other content on this particular page other than the intro paragraph. I believe there are other pages similar to this scenario where there is little content and major links exist, but I reached out to @bethpotts and the content team to verify.

**Comment by aubreyarcangel** [comment]
@nick
here are a few more pages with the same issue thanks to @DanielleSoCompany : 

https://www.va.gov/life-insurance/options-eligibility/
https://www.va.gov/careers-employment/vocational-rehabilitation/
https://www.va.gov/careers-employment/vocational-rehabilitation/programs/

**Comment by ncksllvn** [comment]
Ah I see @aubreyarcangel. In Metalsmith, I believe we have a `content` property available to us, and that's the actual content of the Markdown files (everything below the front matter.)  I wonder if we can check if `content` is empty, and remove the padding if so. The `content` may actually be set to whitespace (spaces and newlines) so the [strip](https://shopify.github.io/liquid/filters/strip/) filter may be handy.

File: src/site/includes/detail-page.html [hunk 1]
```diff
@@ -20,10 +20,18 @@
...
+        <h1>{{ heading }}</h1>
+        {{ contents }}
+      </article>
+    {% else %}  
       <article class="usa-content">
         <h1>{{ heading }}</h1>
         {{ contents }}
       </article>
+    {% endif %}
+
```
Comment (ncksllvn) [inline src/site/includes/detail-page.html]: Liquid actually doesn't have triple equals. I think this evaluates to `true` every time, so I think you'll need to switch to double equals.

**Review by ncksllvn — APPROVED** [review]

File: src/site/includes/detail-page.html (hunk 1 above)
Comment (aubreyarcangel) [reply src/site/includes/detail-page.html]: Thanks! I removed `== true`

**Pull request merged**